packages/p2p/src/components/__tests__/app-content.spec.js

```diff
@@ -39,6 +45,10 @@ describe('<AppContent/>', () => {
         useStores.mockImplementation(() => ({
             general_store: mocked_store_values,
         }));
+        useModalManagerContext.mockImplementation(() => ({
+            showModal: () => {},
+            hideModal: () => {},
```

Comment:
Added mock function for `useModalManagerContext` since test is not passing as it was unable to identify the context hook for modal manager:
![Screenshot 2022-12-13 at 11 44 08 AM](https://user-images.githubusercontent.com/103016120/207222808-0781648a-2f13-4e57-8ea3-581a69d27e08.png)